interceptors/logging/options.go

```diff
@@ -204,3 +206,10 @@ func WithTimestampFormat(format string) Option {
 		o.timestampFormat = format
 	}
 }
+
+// WithDisableLoggingFields disables logging of gRPC fields provided.
```

Comment:
Add some reference to where the default logging fields are defined, and also add an example use case here please.